simtools/configuration/commandline_parser.py

```diff
@@ -288,3 +288,32 @@ def efficiency_interval(value):
             raise argparse.ArgumentTypeError(f"{value} outside of allowed [0,1] interval")
 
         return fvalue
+
+    @staticmethod
+    def zenith_angle(angle):
+        """
+        Argument parser type to check that the zenith angle provided is in the interval [0, 180].
+        We allow here zenith angles larger than 90 degrees in the improbable case
+        such simulations are requested. It is not guaranteed that the actual simulation software
+        supports such angles!
+
+        Parameters
+        ----------
+        angle: float
+            zenith angle to verify
+
```

Comment:
add Returns here

Reply:
Fixed (also the other cases in this file).